packages/exporter-trace-otlp-http/src/platform/node/util.ts

```diff
@@ -42,7 +42,19 @@ export function sendWithHttp<ExportItem, ServiceRequest>(
   onSuccess: () => void,
   onError: (error: otlpTypes.OTLPExporterError) => void
 ): void {
+  const exporterTimeout = collector.timeoutMillis;
   const parsedUrl = new url.URL(collector.url);
+  let reqIsDestroyed: boolean;
+  const nodeVersion = Number(process.versions.node.split('.')[0]);
+
+  const exporterTimer = setTimeout(() => {
+    reqIsDestroyed = true;
+    if (Number(nodeVersion) >= 14) {
```

Comment:
```suggestion
    if (nodeVersion >= 14) {
```
No need for conversion here as it's already done during parsing